### PR TITLE
homework for lesson 3

### PR DESCRIPTION
**Pull request by SpaceChihuahua**

### CODE REVIEW

File: lesson3/lesson3.cpp [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+{
+    //First task
+    int Time = 0;
+    int Hours = 0;
+    int Minutes = 0;
+    int Seconds = 0;
+    std::cout << "Enter how many time you spent in game in seconds: ";
+    std::cin >> Time;
+
+    Hours = Time / 3600;
```
Comment (bohdanpc) [inline lesson3/lesson3.cpp:14]: Consider using const variables for const values:
const unsigned SecondsPerMinute = 60;
const unsigned MinutesPerHour = 60;
const unsigned SecondsPerHour = SecondsPerMinute * MinutesPerHour;

File: lesson3/lesson3.cpp [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    }
+    else
+    {
+        std::cout << "If you entered negative numbers, I assume you meant positive ones, so here are your calculations \n";
+        std::cout << "Perimeter is: " << 2 * Width + 2 * Height << std::endl;
+        std::cout << "Square is: " << Width * Height << std::endl;
+    }
+
+    //fifth task
+    double pi = 2 * acos(0.0);
```
Comment (bohdanpc) [inline lesson3/lesson3.cpp:75]: trigonometric functions can be costly when called frequently. Consider using precalculated value, M_PI from <cmath> or std::numbers::pi from <numbers>

File: lesson3/lesson3.cpp (hunk 1 above)
Comment (SpaceChihuahua) [reply lesson3/lesson3.cpp]: Added const values

File: lesson3/lesson3.cpp (hunk 2 above)
Comment (SpaceChihuahua) [reply lesson3/lesson3.cpp]: Used M_PI and also added calculations for circle circumference (I forgot to do it first time)

**Review by bohdanpc — APPROVED** [review]

**Pull request merged**